.gitignore

```diff
@@ -1,2 +1 @@
 node_modules
-/dist/
```

Comment:
@UgurGumushan Please commit the dist folder and yarn.lock file.

Reply:
[Yes](https://github.com/kontist/validate-steuerid/pull/3/commits/39300d0ae4e4faa371cd4b6a6d4c0615e5920c75)